xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3227,6 +3227,13 @@ void CVideoPlayer::UpdateStreamInfos()
     s.stereo_mode = m_VideoPlayerVideo->GetStereoMode();
     if (s.stereo_mode == "mono")
       s.stereo_mode = "";
+
+    CDemuxStream* stream = m_pDemuxer->GetStreamFromVideoId(streamId);
+    if (stream && stream->type == STREAM_VIDEO)
```

Comment:
I would rather use m_pDemuxer->GetStream(m_CurrentVideo.id)
GetStreamFromVideoId is used nowhere in the code and should be removed.